random-fu/random-fu.cabal

```diff
@@ -1,5 +1,5 @@
 name:                   random-fu
-version:                0.2.7.4
+version:                0.2.7.5
```

Comment:
And this should be at least `0.2.7.7`, see [hackage](https://hackage.haskell.org/package/random-fu)

Reply:
👍